grudge/dt_utils.py

```diff
@@ -271,25 +271,33 @@ def dt_geometric_factors(
     dd_face = DOFDesc("all_faces", dd.discretization_tag)
     face_discr = dcoll.discr_from_dd(dd_face)
 
-    # To get a single value for the total surface area of a cell, we
-    # take the sum over the averaged face areas on each face.
-    # NOTE: The face areas are the *same* at each face nodal location.
-    # This assumes there are the *same* number of face nodes on each face.
```

Comment:
The updated code no longer describes this implicit assumption, which AFAICT is still being made.

* Why is this assumption needed in the first place?
* Could it be checked using code?

Reply:
Clean up in https://github.com/inducer/grudge/pull/195/commits/4f04de2e48b829aca1050e298aaf5d6d7a75237e